azure/scope/managedcontrolplane.go

```diff
@@ -213,6 +215,11 @@ func (s *ManagedControlPlaneScope) AdditionalTags() infrav1.Tags {
 	return tags
 }
 
+// AzureFleet returns the cluster AzureFleet.
+func (s *ManagedControlPlaneScope) AzureFleet() *infrav1.FleetManager {
```

Comment:
should this func be consistent in the naming with what it returns? why AzureFleet() and not FleetManager()?
other funcs in this file do not prefix with `Azure`

Reply:
Good catch!